cli/src/description_util.rs

```diff
@@ -60,8 +60,13 @@ pub fn edit_multiple_descriptions(
     let mut bulk_message = String::new();
 
     bulk_message.push_str(indoc! {r#"
-    JJ: Enter or edit commit descriptions after the `JJ: describe` lines.
-    
+        JJ: Enter or edit commit descriptions after the `JJ: describe` lines.
+
+        JJ: Warning: Describing multiple commits at once is currently
+        JJ: **experimental**.
+        JJ: - The text you enter will be lost on a syntax error.
+        JJ: - The syntax may change in the future.
```

Comment:
How is this? I toned it down a little.

I would still like to mention that the syntax may change, because 1) I would like it to change and 2) I don't want people parse or create this syntax in tools like `gg` or custom editors without being aware that the ground is likely to shift under them. That said, if people prefer, I can remove it; it's not *that* important.

Reply:
I'm okay with the updated comment, but I would omit the first warning line. (It's not important to user whether this feature is under development or not.)
```
        JJ: Enter or edit commit descriptions after the `JJ: describe` lines.
        JJ: - The text you enter will be lost on a syntax error.
        JJ: - The syntax of separate lines may change in the future.
```

Reply:
I can make that line just be `JJ: Warning:` or `JJ: Note:`. Without anything, the topic changes too abruptly IMO.